README.md

```diff
@@ -1,10 +1,10 @@
-# Vulkan Ecosystem Components
+# Vulkan SC Ecosystem Components
 
-This project provides the Khronos official Vulkan ICD desktop loader for Windows, Linux, and MacOS.
+This project provides the Khronos Vulkan SC Development ICD loader for Windows, Linux, and QNX.
```

Comment:
Should we add anything like, "and also provides the vulkan loader?" since we can build both the Vulkan and Vulkan SC loaders via this project?

Reply:
added